### ISSUE TITLE
Default installation with all plugins

### ISSUE THREAD

**Issue by hdijkema** [syncany/syncany#356]
I think it would be a good idea to distribute an installer that defaults to the syncany GUI and with all released plugins activated. This would be a good step in the line of making syncany more end-user oriënted.

**Comment by cr0** [comment]
If we do that the installer should allow to select the plugins which are going to be installed.

**Comment by hdijkema** [comment]
Maybe. However, why? Most end-users do not have the technical knowledge.

Hans Dijkema
Weerdslag 64
7206BT Zutphen

telefoon: 06 4639 0827
email: hans@dykema.nl <mailto:hans@dykema.nl>
Zakelijk: hans.dijkema@covalent.nl <mailto:hans.dijkema@covalent.nl>

Christian Roth schreef op 27-1-2015 om 10:25:

**Comment by pimotte** [comment]
The problem with including all plugins is that this easily adds another 10MB to the distributable size, while most users will be using one, maybe two plugins.

**Comment by hdijkema** [comment]
It depends what you want to optimize. Size <-> usability. With todays harddisk sizes and internet connections 10MB extra would not be very much.

**Comment by binwiederhier** [comment]
I don't want to be the bad guy here, but to be honest, I have my reservations to make a package with all the plugins. Here are my top 3 reasons:

- I don't think that there is any use case in which people will need all the plugins. 
- Although Syncany tries to be very easy to use, people need to bring their own storage, so it's safe to assume they have at least some technical knowledge. If they just get a syncany:// link, then Syncany should ask to install that plugin if it is not installed yet (Do we do that? I don't think so!) 
- Plugins have been known to conflict with one another. Some plugins use the same libraries in different versions, We try to notice when something conflicts and add them to the "Plugin-Conflicts-With" metadata field, but it'd be a pain to keep track of that with every release.

In conclusion, I think don't think it's necessary at this point. I hope that's alright with everyone.

**Comment by hdijkema** [comment]
I think these considerations are certainly important and valid.
